test/test_models.py

```diff
@@ -55,7 +53,7 @@ def check_script(self, model, name):
             tb = traceback.format_exc()
             scriptable = False
             msg = str(e) + str(tb)
-        self.assertEqual(torchub_models[name], scriptable, msg)
+        self.assertEqual(scriptable, scriptable, msg)
```

Comment:
I'm not sure this change is what you want to do?

Reply:
Since they are all scriptable now, we don't need to access `torchub_models` and just assert that `scriptable` is True

Reply:
Can you then make
```python
self.assertEqual(scriptable, True, msg)
```

Reply:
oh oops - good catch